src/app/search/search.component.html

```diff
@@ -20,3 +20,13 @@ <h1>Select an Artifact</h1>
     </div>
   </div>
 </div>
+
+<div class="showNotification" *ngIf="errorMsg">
```

Comment:
The div shouldn't be necessary. You should be able to target the pfng-toast-notification element directly.